src/forms/handler.js

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2024 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+import RUMAPIClient from '@adobe/spacecat-shared-rum-api-client';
+import { getRUMDomainkey, getRUMUrl } from '../support/utils.js';
+import { AuditBuilder } from '../common/audit-builder.js';
+import { wwwUrlResolver } from '../common/audit.js';
+
+const DAILY_THRESHOLD = 1000;
```

Comment:
Daily_Threshold 1000 is too big for now. We have to tweak it to come up with a better number

Reply:
based on surest, I'd suggest to keep it 200.

Reply:
keeping it 200 for now